assets/js/lib/test-utils/factories/sapSystems.js

```diff
@@ -26,26 +26,25 @@ export const sapSystemApplicationInstanceFactory = Factory.define(() => ({
   https_port: faker.internet.port(),
   instance_hostname: faker.hacker.noun(),
   instance_number: faker.number.int({ min: 10, max: 99 }).toString(),
-  sid: faker.string.alphanumeric(3, { casing: 'upper' }),
+  sid: generateSid(),
   start_priority: faker.number.int({ min: 1, max: 9 }).toString(),
   sap_system_id: faker.string.uuid(),
   absent_at: null,
 }));
 
 export const sapSystemFactory = Factory.define(({ params }) => {
   const id = params.id || faker.string.uuid();
-  const sid = params.sid || faker.string.alphanumeric(3, { casing: 'upper' });
+  const sid = params.sid || generateSid();
   const databaseId = params.database_id || faker.string.uuid();
-  const databaseSid =
-    params.database_sid || faker.string.alphanumeric(3, { casing: 'upper' });
+  const databaseSid = params.database_sid || generateSid();
   return {
     application_instances: sapSystemApplicationInstanceFactory.buildList(2, {
       sap_system_id: id,
       sid,
     }),
     database_instances: databaseInstanceFactory.buildList(2, {
       database_id: databaseId,
-      sid: faker.string.alphanumeric(3, { casing: 'upper' }),
+      sid: generateSid(),
```

Comment:
Shouldn't this and the next one be `databaseSid`, generated on top?
I mean, a SAP system all database instances should have the same SID, right?

Reply:
Yeah, you are fully right with this!